### PR TITLE
UN SDG: Making garden tables unique and ensuring source name is unique

### PR DESCRIPTION
**Pull request by spoonerf**
* Previously used just the series code as the Garden table short name but have changed this to `indicator_seriesCode` as some series codes are shared across indicators.
* Ensuring Source.name is unique so that Source.description can vary when Data Publisher's source is the same.

### CODE REVIEW

**Comment by spoonerf** [comment]
Moved the source description into the variable description as this doesn't face the same constraints.

File: etl/steps/grapher/un_sdg/2022-07-07/output/variable_replacements.json [hunk 1]
```diff
@@ -1,297 +1,155 @@
...
-    "417077": "431304",
-    "417079": "429400",
-    "417082": "429370",
-    "417100": "430041",
-    "417159": "430042",
-    "417272": "430121",
-    "417273": "430122",
-    "417421": "431708"
-}
+    "414079": "456880",
```
Comment (Marigold) [inline etl/steps/grapher/un_sdg/2022-07-07/output/variable_replacements.json:2]: (The list got shorter, just checking whether this is expected)

Comment (spoonerf) [reply etl/steps/grapher/un_sdg/2022-07-07/output/variable_replacements.json:2]: Ah yeh I think that's okay - the variable matching and chart revisions is still a manual step. I wonder if we should add it as a dag step or if that is overkill, I don't think we would want it run every time but maybe we don't want manual bits of code hanging around in the etl?

Comment (Marigold) [reply etl/steps/grapher/un_sdg/2022-07-07/output/variable_replacements.json:2]: Hmm, interesting idea. Maybe not exactly a dag step, but some helper command would be helpful.

I really like your `match_vars_crs.py` script for simplicity though. If we decide to automate it, I'd just steal it anyway.

Comment (spoonerf) [reply etl/steps/grapher/un_sdg/2022-07-07/output/variable_replacements.json:2]: Yeh maybe an optional flag would be cool! okay will merge this now :)

**Pull request merged**